st2common/st2common/models/api/action.py

```diff
@@ -148,6 +149,11 @@ class ActionAPI(BaseAPI):
                 "patternProperties": {
                     "^\w+$": util_schema.get_draft_schema()
                 }
+            },
+            "tags": {
+                "description": "User associated metadata assigned to this object.",
```

Comment:
I prefer `tags` as well. It makes it relatively simple to group things together in a Many-Many type relationship.

I'm also on the same page of KISS here. Small limited entries here.

![voted](https://cloud.githubusercontent.com/assets/20028/5635259/6b1b34f0-959c-11e4-854f-2ac7780d23d1.png)